app/lib/datadog_api.rb

```diff
@@ -0,0 +1,48 @@
+require 'dogapi'
+
+module DatadogApi
+
+  METRIC_TYPES = {
+      count: "count".freeze,
+      gauge: "gauge".freeze,
+      rate: "rate".freeze,
+  }.freeze
+
+  class Configuration
+    attr_accessor :enabled, :env, :api_key, :namespace
+  end
+
+  class << self
+    attr_writer :configuration
+  end
+
+  def self.configuration
+    @configuration ||= Configuration.new
+  end
+
+  def self.configure
+    yield configuration
+  end
+
+  def self.client
+    @statsd ||= Dogapi::Client.new(configuration.api_key)
+  end
```

Comment:
Since we are using the API client and not DogStatsD, I might change the name of the variable here. Not a big deal.

Reply:
oops you're right good call, will change